framework/ui/src/main/java/com/google/android/catalog/framework/ui/CatalogActivity.kt

```diff
@@ -63,13 +57,6 @@ open class CatalogActivity : FragmentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        // Ensure that the declaring activity theme don't show an actionbar
-        actionBar?.hide()
```

Comment:
I think we should keep this? Given that the client could potentially use any theme since it's extending the activity. This ensures the actionBar would be hidden and avoid the double toolbar

Reply:
We can keep this, but we still need to hide the action bar in the theme. Otherwise, `Scaffold` cannot find the correct inset values.